### DISCUSSION TITLE
Programmatically intercepting request responses

### DISCUSSION THREAD

**Discussion by vihanga-w** [ulixee/hero#337] [Q&A] [answered]
Hi, is there any way I can intercept the response to all requests and programmatically filter these maybe through a callback. Would be good if the response headers are also available.

Thanks

**Comment by blakebyrnes** [comment, marked as answer]
Yes, but if you want to modify responses, you need to write a plugin (https://ulixee.org/docs/hero/plugins/core-plugins)

**Reply by vihanga-w** [reply]
ok thanks
